draft-ietf-quic-transport.md

```diff
@@ -1721,7 +1721,12 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 ## Path Validation Responses
 
 On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
-echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame,
+unless it has PATH_RESPONSE frames buffered for the same destination connection
+ID and wishes to limit memory consumption.
```

Comment:
I don't think this addition is necessary.  QUIC does not require or suggest that PATH_RESPONSE be buffered, so an implementation would simply send or queue subsequent PATH_RESPONSE frames, as it should. Memory considerations are much larger with data, and I would argue that memory pressure due to too many PATH_CHALLENGEs isn't interesting.

Reply:
That's part of what @ianswett was attempting to address when we brought this issue up, but I think the idea there was deliberately about rate limiting of the amount of memory you could be required to consume. Not sure how much text we need there... 
@ianswett, thoughts?

Reply:
If we forbid multiple PATH_CHALLENGE frames in a single packet, I believe we don't need to add text about buffering.  But I'm not sure if that's an acceptable restriction to the WG.  If not, an alternate option would be to say you SHOULD only send one PATH_CHALLENGE in a packet and you SHOULD only respond to one PATH_CHALLENGE from a single packet.

Reply:
I'd be happy with saying SHOULD only send one and SHOULD only respond to one. Generally, I'm not a huge fan of collapsing multiple path challenges, but if we combine that with "don't send something that will get collapsed" then if that happens its your own fault.

Reply:
Added text for this and removed the "buffered" sentence.